libs/net/models/Areas/Admin/Report/ChartTemplateModel.cs

```diff
@@ -1,23 +1,29 @@
 using System.Text.Json;
 using TNO.API.Models;
+using TNO.API.Models.Settings;
 
 namespace TNO.API.Areas.Admin.Models.Report;
 
 /// <summary>
-/// ChartTemplateModel class, provides a model that represents an report template.
+/// ChartTemplateModel class, provides a model that represents an chart template.
 /// </summary>
 public class ChartTemplateModel : BaseTypeWithAuditColumnsModel<int>
 {
     #region Properties
     /// <summary>
-    /// get/set - The Razor template to generate the report.
+    /// get/set - The Razor template to generate the chart.
     /// </summary>
     public string Template { get; set; } = "";
 
     /// <summary>
-    /// get/set - The settings for this report.
+    /// get/set - The settings for this chart template.
     /// </summary>
-    public Dictionary<string, object> Settings { get; set; } = new Dictionary<string, object>();
+    public ChartTemplateSettingsModel Settings { get; set; } = new();
+
+    /// <summary>
+    /// get/set - The settings for this section chart.
+    /// </summary>
+    public ChartSettingsModel? SectionSettings { get; set; }
```

Comment:
We use this model for two related purposes.  To support when it's a many-to-many relationship with a section it now has the `SectionSettings`.